shared/chat/inbox/container.js

```diff
@@ -43,10 +43,11 @@ const getRows = createImmutableEqualSelector([filteredInbox, getPending], (inbox
   return I.List(pending.keys()).concat(inbox)
 })
 
-const mapStateToProps = (state: TypedState) => ({
+const mapStateToProps = (state: TypedState, {isActiveRoute}) => ({
```

Comment:
Note that you don't need to pass `isActiveRoute` down yourself, and this will have no effect. `pausableConnect` includes it in the connected component's props implicitly since `mapStateToProps` will not run if `isActiveRoute` is false.